src/include/byteorder.h

```diff
@@ -53,3 +53,8 @@ inline ceph_les16 init_les16(__s16 x) {
   v = x;
   return v;
 }
+
+template <typename T>
+auto format_as(ceph_le<T> c) {
+  return (T)c;
```

Comment:
```suggestion
  return static_cast<T>(c);
```

probably it's but personal taste, but i'd suggest use a named cast just to be more explicit. see http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts-named .

Reply:
c3e05b5732af6a0722591f59b991178de3042e45